### PR TITLE
placeholder overlay. removed by dom function

### LINKED ISSUE #8: Remove brackets from JSON text block
They are confusing.

### PR DESCRIPTION
**Pull request by SamuelGHill**
Resolves issue #8 

Could not add placeholder within jsoneditor, https://github.com/josdejong/jsoneditor/issues/696, so this was done by DOM manipulation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -198,15 +199,27 @@ <h2>Connect</h2>
       </div>
     </footer>
     </div>
-    
     <script>
       var cb = new ClipboardJS("#copy-button");
+      
+      // Display placeholder and remove on click.
+      var placeholder = {
```
Comment (mtrythall) [inline index.html]: const?

**Review by mtrythall — DISMISSED** [review]
All works. Would be nice if the message came back on blur, but that's a stretch goal. Minor note about usage of `var` which seems incorrect below, but not a biggie if it works.

File: index.html (hunk 1 above)
Comment (SamuelGHill) [reply index.html]: yes, it is supported well, so I changed it below, https://caniuse.com/const.

**Review by mtrythall — APPROVED** [review]

**Review by mtrythall — APPROVED** [review]

**Review by mtrythall — APPROVED** [review]

**Comment by SamuelGHill** [comment]
@Cerchie Can you approve this merge? Michael approved it but it won't allow a merge on his approval...

**Review by Cerchie — APPROVED** [review]

**Pull request merged**